src/Payum/Stripe/StripeCheckoutGatewayFactory.php

```diff
@@ -16,7 +16,7 @@ class StripeCheckoutGatewayFactory extends GatewayFactory
      */
     protected function populateConfig(ArrayObject $config)
     {
-        if (!class_exists('Stripe')) {
```

Comment:
if user installed the old version, this where this class Stripe exists we have to throw an exception. We have to told a user that the minimum required version of stripe is 3.6 and he has to upgrade it too.